webpack/assets/javascripts/react_app/components/VerticalNavbar/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { VerticalNav, Dropdown, Icon, MenuItem } from 'patternfly-react';
+import './verticalnavbar.scss';
+
+class VerticalNavbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.items = [];
+    this.childrenArray = [];
+  }
+  componentWillMount() {
+    this.fetchItems();
+  }
+
+  fetchItems() {
+    if (this.props.data !== undefined) {
+      this.props.data.menu.forEach((menu) => {
+        menu.forEach((item) => {
+          this.childrenArray = [];
+          let count = 0;
+          item.children.forEach((child) => {
+            const childObject = {
+              title:
+                child.type === 'divider' ? `//DIVIDER${count}` : child.name, // until dividers are supported
+              href: child.url ? child.url : 'hosts',
+            };
+            this.childrenArray.push(childObject);
+            count += 1;
+          });
+          const itemObject = {
+            title: item.name,
+            iconClass: item.icon,
+            subItems: this.childrenArray,
+          };
+          this.items.push(itemObject);
+        });
+      });
+    } else this.items = this.props.items;
+  }
+
+  render() {
+    return (
+      <VerticalNav hoverDelay={250} items={this.items} {...this.props}>
```

Comment:
Is there a way that we could add this hoverDelay to Foreman settings? I personally want it to be 0ms because that's how the horizontal nav was. I have a feeling I'm not the only user that wants it to work basically the same as the old nav.

Reply:
The current verticalnav uses the default hover timing, which is even longer than 250ms.
I'll be glad to use 0ms. @ohadlevy